lib/topological_inventory/providers/common/collectors_pool.rb

```diff
@@ -5,8 +5,9 @@ module Providers
     module Common
       class CollectorsPool
         SECRET_FILENAME = "credentials".freeze
+        COLLECTOR_POLL_TIME = ENV['COLLECTOR_POLL_TIME']&.to_i || 300
 
-        def initialize(config_name, metrics, collector_poll_time: 60, thread_pool_size: 2)
+        def initialize(config_name, metrics, collector_poll_time: COLLECTOR_POLL_TIME, thread_pool_size: 2)
```

Comment:
So this didn't run into the `.minutes` issue?  ie it's an integer.

Reply:
Yep, now defaulting to `ENV['COLLECTOR_POLL_TIME'].to_i || 300`

Reply:
Ahh - so the `.minutes` issue was due it being a string before?

Reply:
No - it's because `Integer` doesn't have the `:minutes` method in vanilla ruby, I assume it's an ActiveSupport method or something.